src/components/content/order/migrate/MigrateServiceStatusPolling.tsx

```diff
@@ -0,0 +1,150 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * SPDX-FileCopyrightText: Huawei Inc.
+ */
+
+import { useStopwatch } from 'react-timer-hook';
+import React, { useEffect } from 'react';
+import { useServiceDetailsPollingQuery } from '../orderStatus/useServiceDetailsPollingQuery';
+import { ServiceDetailVo } from '../../../../xpanse-api/generated';
+import { OrderSubmitResult } from '../orderStatus/OrderSubmitResult';
+import { OrderSubmitFailed } from '../orderStatus/OrderSubmitFailed';
+import { ProcessingStatus } from '../orderStatus/ProcessingStatus';
+import { OperationType } from '../formElements/CommonTypes';
+
+function MigrateServiceStatusPolling({
+    uuid,
+    error,
+    isLoading,
+    setIsDestroying,
+    setIsDestroyingCompleted,
+}: {
+    uuid: string;
+    error: Error | undefined;
+    isLoading: boolean;
+    setIsDestroying: (arg: boolean) => void;
+    setIsDestroyingCompleted: (arg: boolean) => void;
+}): React.JSX.Element {
+    const getDeployedServiceDetailsByIdQuery = useServiceDetailsPollingQuery(uuid, [
```

Comment:
also.. can you please write one comment in the code.. on why we poll only for destroy status?